rdr_service/api/nph_participant_api_schemas/schema.py

```diff
@@ -217,7 +217,7 @@ class Participant(ObjectType):
             from the AoU participant_summary table’s suspension columns
         '''
     )
-    enrollmentStatus = SortableField(
+    nphEnrollmentStatus = SortableField(
```

Comment:
The name of this field is still aouEnrollmentStatus. And the description field describes AoU enrollment statuses. Should the `nph_enrollment_status` field's name (down on line 266) been changed instead?

Can you also please add a unit test to verify this change?